docs/reference/pip_download.rst

```diff
@@ -0,0 +1,41 @@
+
+.. _`pip download`:
+
+pip download
+------------
+
+.. contents::
+
+Usage
+*****
+
+.. pip-command-usage:: download
+
+``pip download`` is used just like ``pip install --download`` was used in the past. The same rules apply to this command: it should be very comfortable to use.
```

Comment:
Clarified the documentation.  I noted explicitly that this replaces `pip install --download`, but that the latter is deprecated and will be removed.  I'm not very familiar with this doc formatting, so if there's any extra cross-linking I should be doing to `pip install`, please point me at an example of how to do so.